src/loaders/env.js

```diff
@@ -0,0 +1,3 @@
+import dotenv from 'dotenv';
+
+export default dotenv.config({ silent: true });
```

Comment:
This should run before all other code that might use `process.env`, hence it is important to export it in the index.js file before all other exports.

For more info, please see here: https://www.npmjs.com/package/dotenv#how-do-i-use-dotenv-with-import- and here: https://github.com/motdotla/dotenv/issues/133#issuecomment-255298822